.github/workflows/release-windows.yml

```diff
@@ -0,0 +1,63 @@
+on:
+    release:
+        types: [published]
+    workflow_dispatch:
+
+name: Release Windows
+
+jobs:
+    build:
+        runs-on: windows-2022
+        permissions:
+            contents: write # needed for uploading release artifact
+        defaults:
+            run:
+                shell: msys2 {0}
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+            - name: Set installer name
+              id: set_installer_name
+              shell: bash
+              run: echo "name=rnote-win-installer-$(echo ${GITHUB_REF_NAME#v})-x86_64" >> $GITHUB_OUTPUT
+            - name: Setup MSYS2
+              uses: msys2/setup-msys2@v2
+              with:
+                  release: false
+                  update: true
+                  install: |
+                      git mingw-w64-x86_64-xz mingw-w64-x86_64-pkgconf mingw-w64-x86_64-gcc mingw-w64-x86_64-clang mingw-w64-x86_64-toolchain
+                      mingw-w64-x86_64-autotools mingw-w64-x86_64-make mingw-w64-x86_64-cmake mingw-w64-x86_64-meson mingw-w64-x86_64-diffutils
+                      mingw-w64-x86_64-desktop-file-utils mingw-w64-x86_64-appstream-glib mingw-w64-x86_64-gtk4 mingw-w64-x86_64-libadwaita
+                      mingw-w64-x86_64-poppler mingw-w64-x86_64-poppler-data
+            - name: Remove libpthread.dll.a
+              run: rm /mingw64/lib/libpthread.dll.a
+              continue-on-error: true
```

Comment:
should we continue on error here? I believe the linker then will always fail.

Reply:
I'm not sure, my thought process here was that this might be fixed in MSYS one day, and that the job shouldn't fail in that case. This file being removed or renamed isn't really a possibility as far as I can tell.

Reply:
file permissions could also be a source of error. but valid points so lets leave as is